CodeFirst/Assement.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace TechnicalAssesment
+{
+    public enum GameResult
+    {
+        GameNotStarted,
+        DealerWon,
+        PlayerWon,
+        Tie,
+    }
+
+    public class ScoreBoard
+    {
+        public byte playerWins;
+        public int DealerWins;
+        public byte Ties { get; set; }
+    }
+
+    public class Deck
+    {
+        public List<int> cards;
```

Comment:
Inconstancy in first letter casing. EG: public List<int> cards -->public List<int> Cards